client/my-sites/plugins/plugins-list/index.jsx

```diff
@@ -131,12 +131,12 @@ export default React.createClass( {
 		const bulkManagement = ! this.state.bulkManagement;
 
 		if ( bulkManagement ) {
+			this.setBulkSelectionState( this.props.plugins, bulkManagement );
```

Comment:
Why do we pass `bulkManagement` as a second argument here? What is the semantics of the second argument of `setBulkSelectionState()`?